### PR TITLE
Add an is_unix key

### PR DESCRIPTION
**Pull request by mato**

### CODE REVIEW

**Comment by yomimono** [comment]
This looks good to me.  Do you plan on following with another PR to remove `is_xen`?

**Comment by mato** [comment]
Removing is_xen would mean an API break for anyone using it, right? Do we just spring that on people outright, or is there some deprecation mechanism we can use?

File: lib/mirage_key.mli [hunk 1]
```diff
@@ -38,6 +38,9 @@ val pp_target: [ `Unix | `Xen | `MacOSX | `Virtio | `Ukvm ] Fmt.t
 val is_xen: bool value
 (** Is true iff the {!target} keys takes the value [`Xen]. *)
 
+val is_unix: bool value
+(** Is true iff the {!target} key is a UNIXish system. *)
```
Comment (Drup) [inline lib/mirage_key.mli]: Maybe provide the exact list of "UNIXish systems" ?

**Comment by Drup** [comment]
I think you could just keep the usage of `(value target)` inside mirage itself. Probably more future-proof (forcing people who add targets to look at all the cases).

**Pull request merged**